templates/coterie/show.html.twig

```diff
@@ -1,29 +1,20 @@
 {% extends 'base.html.twig' %}
 
-{% block title %}Coterie Details {% endblock %}
+{% block title %}{{ coterie.label }} {% endblock %}
+
+{% block admin %}
+    {% include 'partial/toolbar.html.twig' with {'edit': 'coterie_edit', 'delete': 'coterie_delete', 'id': coterie.id } %}
+{% endblock %}
+
+{% block breadcrumb %}
+    {% include 'partial/breadcrumb.html.twig' with {'index': 'coterie', 'show': coterie.label} %}
+{% endblock %}
 
  {% block pageheader %}
-     <h1>Coterie Details</h1>
+     <h1><span class="firstLine">{{ coterie.label }}</span></h1>
```

Comment:
This should just be an h1, not a firstLine: firstLine is specific to the Poems.

Reply:
Ooh nice catch on that copy/pasta.